codegen.ts

```diff
@@ -0,0 +1,19 @@
+import type { CodegenConfig } from "@graphql-codegen/cli";
+
+const config: CodegenConfig = {
```

Comment:
Introduces codegen to be able to resolver typing support - will add type tests in a future PR